inc/classes/class-offline-form.php

```diff
@@ -105,9 +117,8 @@ private function get_form_urls() {
 		$string      = '';
 		$form_routes = get_option( 'rt_pwa_extension_options' );
 		if ( empty( $form_routes ) || ctype_space( $form_routes ) ) {
-			// Generate random string if no URLs specified.
-			// Random string ensures that no URL match.
-			$string = wp_generate_password( '30', false, false );
+			// Return false if no offline form routes are set.
+			$string = false;
```

Comment:
@deepaklalwani97 Please check if returning false is not causing reloading issue with like the previous method. Also, check if returning blank string works or not. If that works fine then we should use blank string instead of false.

Reply:
@sagarnasit Using empty string does not create reload issue but throws a workbox error which might create issues with other scripts. So I think it would be better to return false.
![Screenshot 2020-03-23 at 8 52 29 PM](https://user-images.githubusercontent.com/32839217/77332767-7cf93080-6d48-11ea-9072-d7d85265ffcb.png)